kaleido/privatestack_bridge_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018 Kaleido, a ConsenSys business
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package kaleido
+
+import (
+	"testing"
+
+	gock "gopkg.in/h2non/gock.v1"
```

Comment:
glad to see we take this approach rather than trying to mock the innards of resty